docs/integrations/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Integrations Overview | bluefox.email documentation
+description: Explore BlueFox Email integrations for enhanced email control and real-time event tracking.
+head:
+  - - meta
+    - name: description
+      content: Explore BlueFox Email integrations for enhanced email control and real-time event tracking.
+  - - meta
+    - property: og:title
+      content: Integrations Overview | bluefox.email documentation
+  - - meta
+    - property: og:description
+      content: Explore BlueFox Email integrations for enhanced email control and real-time event tracking.
+  - - meta
+    - property: og:image
+      content: https://bluefox.email/assets/docs-share.png
+  - - meta
+    - property: og:url
+      content: https://bluefox.email/docs/integrations
+  - - meta
+    - property: og:type
+      content: website
+  - - meta
+    - property: twitter:card
+      content: summary_large_image
+  - - meta
+    - property: twitter:title
+      content: Integrations Overview | bluefox.email documentation
+  - - meta
+    - property: twitter:description
+      content: Explore BlueFox Email integrations for enhanced email control and real-time event tracking.
+  - - meta
+    - property: twitter:image
+      content: https://bluefox.email/assets/docs-share.png
+---
+
+# The bluefox.email integrations
+
+**bluefox.email integrations** connect with various platforms to enhance email delivery, authentication, and real-time tracking. These integrations allow you to send authentication emails, notifications, and monitor email performance with detailed insights. Below are the key integration options:  
```

Comment:
Don't start it with a lowercase letter... if the sentence starts with bluefox, then we have no other choice than using Bluefox.email